core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala

```diff
@@ -78,8 +78,11 @@ trait TestDriverComponent extends DriverComponent with DataAccess with DefaultIn
   val wdlParser = new CachingWDLParser(wdlParserConfig, mockCromwellSwaggerClient)
   val methodConfigResolver = new MethodConfigResolver(wdlParser)
 
+  // TODO: can we be any more targeted about inTransaction vs. inTransactionWithAttrTempTable here?
+  // this is used by many tests to set up fixture data, which includes saving entities, therefore it needs
```

Comment:
test runtime seems to be in line with what it is on develop, but I will leave this TODO here